### DISCUSSION TITLE
Github pages wizard, where is the theme chooser?

### DISCUSSION THREAD

**Discussion by tzaeschke** [community/community#32750] [Other Feature Feedback, Questions, & Ideas] [answered]
I am trying to create a new github pages website using the wizard: https://pages.github.com/
I clicked on "Project Site" and created a new repository.
Then I tried to follow the instructions:

1) "Click on the setting tab" -> Done.
2) "Scroll down to the GitHub Pages section." and " Press Choose a theme."
    -> I cannot do that, my settings page ("General") does not contains any GitHub Pages section. Here is a screenshot of the upper part, and yes, I scrolled down, but there was nothing with "pages":
    
![image](https://user-images.githubusercontent.com/1595847/189628767-b2fc21ee-4d69-411b-a870-0efb208eead0.png)

On the left side there is an item "Pages". This shows some stuff related to GitHub pages, but still no theme chooser.   

Where can I find the theme chooser?    

Bonus question: It seems there is "Github Pages" and also an optional Jekyll integration. There are tons of documentation on Jekyll, but what I could not find is the following: Why would/should I decide to use Jekyll or instead simply GitHub pages? 
All I want is a simple website with text, some graphics and links. No UI effects, forms or dynamic content.

**Comment by puglet5** [comment]
Hey there! Please, refer to these [docs](https://docs.github.com/en/pages) on setting up GitHub Pages. There's also a [section](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll) specifically on Jekyll, it's themes with a ton of useful links. 

Use Jekyll if you want to generate your pages from templates. AFAIK GitHub pages just serves the content from your repository (so you can use something like [Docsify](https://github.com/docsifyjs/docsify) and then let GH Pages serve the generated content).

**Reply by tzaeschke** [reply]
Thanks for your answer. Unfortunately this doesn't quite solve the problem.
- Why don't is see the theme chooser? I.e. is there something wrong with my repository or is the wizard outdated? 

I would be nice if I could simply follow the instructions provided by the GitHub pages wizard. That would save me from reading a ton of documentation. Or is the wizard known to be broken/outdated?

**Comment by thuongtruong1009** [comment, marked as answer]
Hi @tzaeschke 
I using jekyll-theme-cayman at [my-repository](https://github.com/thuongtruong1009/Snake-Game-OOP)
It is simply when you:
+ add _config.yml file: ```theme: jekyll-theme-cayman```
+ Click setting > Pages and choose to deploy from the branch option as below picture. Reference at [Github Page](https://docs.github.com/en/pages)
![image](https://user-images.githubusercontent.com/71834167/189711027-a16083a5-fe7a-4cfd-8449-4de9d2297147.png)

**Reply by tzaeschke** [reply]
Thanks for this example (I have looked for "simple example" but yours is truly simple)! Exactly what I was looking for :-D .
It doesn't answer though why I can't see the theme chooser, but maybe to documentation is just outdated.
I'll try this out and flag it as answer when I got it working.

**Reply by briankissler** [reply]
I've tried this on my page just shows up as blank.  I made the change to the yml file, committed to branch and merge requested.  Any suggestions? 

<img width="666" alt="image" src="https://user-images.githubusercontent.com/55636163/208251888-85076034-f0d5-472a-836a-3fa8034e065a.png">

**Reply by kekeedme** [reply]
Hello @tzaeschke I had the same problem, and found out that they deprecated theme chooser, without telling anyone, and without updating their documentation page. They also claim that they had not updated the documentation yet because "more is coming".... But it's been a while and also, lots of people don't want to learn how to use jekyll on top of other things they need to do, so the community has been telling them it was a bad idea but.... You can see it here, https://github.com/orgs/community/discussions/33924

**Reply by tzaeschke** [reply]
@kekeedme Thanks for pointing that out. And yeah, they could at least have updated the docs, instead of letting people spend time on trying to debug when it doesn't work....

**Reply by kekeedme** [reply]
My pleasure. I wanted to share to, at least, try and help others and prevent them for experiencing the same frustration we have haha

**Comment by CalebZombie21** [comment]
i had the same problem

**Comment by mks10001** [comment]
没用主题选择按钮
![没用主题选择按钮](https://github.com/user-attachments/assets/1cc2cb70-20e6-47b8-b9af-2514fa472344)
